pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1622,6 +1623,17 @@ var _ = Describe("Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(domSpec).ToNot(BeNil())
 		})
+
+		Context("on call to InterfacesStatus", func() {
+			It("should return nil when no interfaces exists", func() {
+				agentStore := agentpoller.NewAsyncAgentStore()
+				manager, _ := NewLibvirtDomainManager(mockConn, testVirtShareDir, nil, 0, &agentStore, "/usr/share/OVMF")
```

Comment:
This seems to fit the setup fixture of the test, so it will be nice to see it in a `BeforeEach`.